autode/neb/original.py

```diff
@@ -64,7 +107,7 @@ def total_energy(flat_coords, images, method, n_cores):
     return sum(rel_energies)
 
 
-def derivative(flat_coords, images, method, n_cores):
+def derivative(flat_coords, images, method, n_cores, plot_energies):
```

Comment:
this doesn;t get used?

Reply:
I assume its for some scipy related reason, that the jacobian needs the same args as the energy function?

Reply:
exactly the latter - will add a comment